### PR TITLE
Improve windows build instructions.

### PR DESCRIPTION
**Pull request by MForster**
Add a command line that installs all required Visual Studio components
as well as Git and Python.

### CODE REVIEW

**Comment by MForster** [comment]
@swift-ci Please test

File: docs/WindowsBuild.md [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
+  --add Component.CPython3.x64 ^
+  --add Microsoft.VisualStudio.Component.Git ^
+  --add Microsoft.VisualStudio.Component.VC.ATL ^
+  --add Microsoft.VisualStudio.Component.VC.CMake.Project ^
+  --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 ^
+  --add Microsoft.VisualStudio.Component.Windows10SDK ^
+  --add Microsoft.VisualStudio.Component.Windows10SDK.17763
+```
+
+If you prefer you can install everything individually, but make sure to include "Programming Languages|Visual C++" and "Windows and Web Development|Universal Windows App Development|Windows SDK" in your installation. The components listed above are required.
```
Comment (martinboehme) [inline docs/WindowsBuild.md:25]: Maybe "by hand" instead of "individually"?

**Review by martinboehme — APPROVED** [review]
This is great -- one less thing to go wrong!

**Comment by MForster** [comment]
@swift-ci Please test

File: docs/WindowsBuild.md [hunk 2]
```diff
@@ -1,23 +1,34 @@
...
+vs_community ^
+  --add Component.CPython2.x86 ^
+  --add Component.CPython3.x64 ^
+  --add Microsoft.VisualStudio.Component.Git ^
+  --add Microsoft.VisualStudio.Component.VC.ATL ^
+  --add Microsoft.VisualStudio.Component.VC.CMake.Project ^
+  --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 ^
+  --add Microsoft.VisualStudio.Component.Windows10SDK ^
+  --add Microsoft.VisualStudio.Component.Windows10SDK.17763
+```
```
Comment (compnerd) [inline docs/WindowsBuild.md:23]: Hmm, I’d tried that but it seemed to exit and run in the background.  Did that not happen for you?

Comment (MForster) [reply docs/WindowsBuild.md:23]: It runs in the background if I specify `-q`, which I didn't. For me it opens the normal installation dialog with all the components preselected.

Comment (MForster) [reply docs/WindowsBuild.md:23]: Well, actually, depends on what you mean with "in the background". The command certainly exits on the command line. But it opens a UI.

**Comment by MForster** [comment]
@swift-ci Please test

File: docs/WindowsBuild.md (hunk 2 above)
Comment (compnerd) [reply docs/WindowsBuild.md:23]: Really?  Okay, nvm then.  I thought that this was batched not interactive.

**Comment by compnerd** [comment]
Sorry @martinboehme @gribozavr - I think that this is useful information so I am going to merge this, but that will require you to rebase your changes :-(.

**Comment by martinboehme** [comment]
> Sorry @martinboehme @gribozavr - I think that this is useful information so I am going to merge this, but that will require you to rebase your changes :-(.

NP, that's the least of my worries. I just need to figure out how to get the CI to pass without any spurious failures.

**Pull request merged**